pythonfmu/fmi2slave.py

```diff
@@ -70,7 +70,7 @@ def to_xml(self, model_options: Dict[str, str] = dict()) -> Element:
         attrib = dict(
             fmiVersion="2.0",
             modelName=self.modelName,
-            guid=f"{self.guid!s}",
+            guid=f"{self.guid}",
```

Comment:
Sure about this one? `self.guid` is of type `uuid1` and not `str`. `!s` should make the conversion.

Reply:
bump @jrbull